esx_service/vmdk_ops.py

```diff
@@ -244,13 +301,18 @@ def createVMDK(vmdk_path, vm_name, vol_name,
 
 
 def cloneVMDK(vm_name, vmdk_path, opts={}, vm_uuid=None, datastore_url=None):
-    logging.info("*** cloneVMDK: %s opts = %s vm_uuid=%s datastore_url=%s", vmdk_path, opts, vm_uuid, datastore_url)
-
     # Get source volume path for cloning
     error_info, tenant_uuid, tenant_name = auth.get_tenant(vm_uuid)
     if error_info:
         return err(error_info)
 
+    readable_vmdk_path = gets_readable_path(vmdk_path)
```

Comment:
why did you move the loggign after get_tenant ? It is the "function entered" log, it should be before other code